src/azure-cli/azure/cli/command_modules/acr/_help.py

```diff
@@ -468,7 +468,7 @@
 examples:
   - name: Queue a run to execute a container command.
     text: >
-        az acr run -r MyRegistry --cmd MyImage /dev/null
+        az acr run -r MyRegistry --cmd '\\$Registry/MyImage' /dev/null
```

Comment:
what does `\\` do, skip `$`? It looks very odd. Then I start to think should we recommend users to use the pattern in the examples?